packages/dev/core/src/Rendering/IBLShadows/iblShadowsAccumulationPass.ts

```diff
@@ -0,0 +1,254 @@
+import { Constants } from "../../Engines/constants";
+import type { AbstractEngine } from "../../Engines/abstractEngine";
+import type { Scene } from "../../scene";
+import { Vector4 } from "../../Maths/math.vector";
+// import { Logger } from "../Misc/logger";
+import "../../Shaders/iblShadowAccumulation.fragment";
+import "../../Shaders/iblShadowDebug.fragment";
+import { PostProcess } from "../../PostProcesses/postProcess";
+import type { PostProcessOptions } from "../../PostProcesses/postProcess";
+import type { Effect } from "../../Materials/effect";
+import { RenderTargetTexture } from "../../Materials/Textures/renderTargetTexture";
+import type { RenderTargetCreationOptions } from "../../Materials/Textures/textureCreationOptions";
+
+/**
+ * This should not be instanciated directly, as it is part of a scene component
+ */
+export class IblShadowsAccumulationPass {

```

Comment:
maybe flag it as _ then? and also as @internal

Reply:
Done